docs/development/local-development.md

```diff
@@ -16,14 +17,14 @@ For local development some dependencies are required:
 - Python `^3.7`
```

Comment:
I noticed we use Python 3.8 in the build GitHub Action:
https://github.com/renovatebot/renovate/blob/6334c03724cb0b17ebd47fa76c24752bfd030fba/.github/workflows/build.yml#L30

Should we update the list of requirements?

```suggestion
- Python `^3.8`
```

Reply:
sure